src/qtt/algorithms/fitting.py

```diff
@@ -154,15 +154,15 @@ def initFermiLinear(x_data, y_data, fig=None):
 
 # %%
 
-def fitFermiLinear(x_data, y_data, verbose=0, fig=None, l=1.16, use_lmfit=False):
+def fitFermiLinear(x_data, y_data, verbose=0, fig=None, lever_arm=1.16, l=None, use_lmfit=False):
     """ Fit data to a Fermi-Linear function
 
     Args:
         x_data (array): independent variable data
         y_data (array): dependent variable data
         verbose (int) : verbosity (0 == silent). Not used
         fig (int) : figure number
-        l (float): leverarm passed to FermiLinear function
```

Comment:
I documented it as a deprecated parameter. It is not clear when this will be deleted. For entire functions we have the `rdeprecated` decorator, which can be used to give a message untill when the function is supported. For single arguments we do not have such a thing.